mycity/mycity/intents/snow_parking_intent.py

```diff
@@ -16,17 +16,17 @@
 DRIVING_DIST = g_maps_utils.DRIVING_DISTANCE_TEXT_KEY
 DRIVING_TIME = g_maps_utils.DRIVING_TIME_TEXT_KEY
 OUTPUT_SPEECH_FORMAT = \
-    ("The closest snow emergency parking location, {Name}, is at "
+    ("The closest snow emergency parking lot, {Name}, is at "
      "{Address}. It is {" + DRIVING_DIST + "} away and should take "
-     "you {" + DRIVING_TIME + "} to drive there. The parking lot has "
+     "you {" + DRIVING_TIME + "} to drive there from home. The lot has "

```

Comment:
Do we know the user is at home? What if they are at the office?

Reply:
Excellent point - I didn't consider they could have given an address that was not where they lived.  Let's delete the "from home".